index.html

```diff
@@ -27,17 +27,119 @@ <h1>I'm Saningo Glad to see you</h1>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-2"></div>
+              <div class="card-desc">
+                <h2>Muilti-Post Stories</h2>
+                <ul>
+                  <li>FACEBOOK</li>
+                  <li>Full stack Dev</li>
+                  <li>2023</li>
+                </ul>
+                <div class="desc">
+                  <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                  <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <button type="button">See Project</button>
+                </div>
+              </div>
+          </div>
 
-          <div class="empty">
+          <div class="card">
+            <div class="card-3"></div>
+              <div class="card-desc">
+                  <div>
+                    <h2>Facebook 360</h2>
+                    <ul>
+                      <li>FACEBOOK</li>
+                      <li>Full stack DEV</li>
+                      <li>2023</li>
+                    </ul>
+                    <div class="desc">
+                      <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                      <button type="button">See Project</button>
+                    </div>
+                  </div>
+              </div>
+          </div>
 
+          <div class="card">
+            <div class="card-4"></div>
+            <div class="card-desc">
+                <div>
+                  <h2>Facebook React</h2>
+                  <ul>
+                    <li>FACEBOOK</li>
+                    <li>Full stack DEV</li>
+                    <li>2023</li>
+                  </ul>
+                  <div class="desc">
+                    <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button type="button">See Project</button>
+                  </div>
+                </div>
+            </div>
           </div>
+
+        </section>
+        <!-- About Me section -->
+        <section class="about-me">
+          <h3>About Me</h3>
+          <p>Hello I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, dont hestatite to contact me.</p>
+           <button type="button">Get my Resume</button>
```

Comment:
You did an amazing job but kindly fix the design of about me section because it is not like in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please follow the design and fix it. Keep rocking 💪 

Here you have a screenshot of how I am seeing about me section:

![image](https://user-images.githubusercontent.com/74506933/153672028-52bfa35b-d77a-496a-8e10-a97eeaae2345.png)

Reply:
Hi @Donard97 could you please clarify this for me.
Kindly fix the background of the first section because when you added the other sections it took like it is background of body. Please fix it

Reply:
- [ ] [REQUIRED] Please note that you need to stick as much as possible to the template design provided as recommended by the previous reviewer. Kindly ensure to use the same font, font-size, font-color, design arrangement and the style.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![8](https://user-images.githubusercontent.com/58692590/153707726-d2569d87-6c5b-4531-874f-00c000cb2857.png) | ![88](https://user-images.githubusercontent.com/58692590/153707738-f99354ee-6a54-4e92-b74f-3b8dc98b7c9b.png)